test/hydration/hydration.test.js

```diff
@@ -6,7 +6,9 @@ import * as path from 'path';
 import { assert, describe, it } from 'vitest';
 import { create_loader, should_update_expected, try_load_config } from '../helpers.js';
 
-import { assert_html_equal } from '../html_equal.js';
+import { setup_html_equal } from '../html_equal.js';
+
+const { htmlEqual } = setup_html_equal();
```

Comment:
without being familiar with this code, the API before seemed nicer where you could just do `assert_html_equal` and now it takes multiple lines. is there a reason to make it multiple calls?

Reply:
>  is there a reason to make it multiple calls?

Before it was mutating the assert object from vitest, I got some failures running the tests with --no-isolate, the cause turned out to be the Object.assign but didn't want to leave another possibility of mutation causing issues down the line.

> without being familiar with this code, the API before seemed nicer where you could just do assert_html_equal

They are created in a closure with options passed into setup_html_equal. Although, that's only ever used in one place so I suppose I could create and export these functions with the default options for the common case.